merkle-distributor/balance-tree.ts

```diff
@@ -0,0 +1,63 @@
+import { UInt } from "@polkadot/types/codec";
+import { TypeRegistry } from "@polkadot/types/create";
+import { decodeAddress, keccakAsU8a } from "@polkadot/util-crypto";
+import MerkleTree from "./merkle-tree";
+
+export default class BalanceTree {
+  private readonly tree: MerkleTree;
+  constructor(accounts: string[]) {
+    this.tree = new MerkleTree(
+      accounts.map((account, index) => {
+        console.log(`Current element is ${account} with index ${index}`);
```

Comment:
Not sure `console.log` should be here in such a form, possible to have a logger function that only log things in dev mode

Reply:
It's only for debugging purpose and I will remove it for now, and let's have some logger like you mentioned in our production code.